File: BarberMe/BarberMe.Database/Models/Service.cs
namespace BarberMe.Database.Models
{
    public class Service
    {
        public int ServiceId { get; set; }

        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public string? ImageUrl { get; set; }

        public decimal DefaultPrice { get; set; }
        public int DefaultDurationMinutes { get; set; }


        public bool IsActive { get; set; } = true;

        public ICollection<BarberService> BarberServices { get; set; } = new List<BarberService>();
    }
}

File: barberme_mobile/lib/models/service.dart
class Service {
  final int id;
  final String name;
  final String? description;
  final double defaultPrice;
  final int defaultDurationMinutes;
  final String? imageUrl;
  final bool isActive;

  const Service({
    required this.id,
    required this.name,
    this.description,
    required this.defaultPrice,
    required this.defaultDurationMinutes,
    this.imageUrl,
    required this.isActive,
  });

  factory Service.fromJson(Map<String, dynamic> json) {
    return Service(
      id: json['id'] as int,
      name: json['name'] as String,
      description: json['description'] as String?,
      defaultPrice:
          (json['defaultPrice'] as num).toDouble(),
      defaultDurationMinutes:
          json['defaultDurationMinutes'] as int,
      imageUrl: json['imageUrl'] as String?,
      isActive: json['isActive'] as bool,
    );
  }
}